activejob/test/cases/logging_test.rb

```diff
@@ -280,8 +280,8 @@ def test_enqueue_retry_logging_on_retry_job
   def test_retry_stopped_logging
     perform_enqueued_jobs do
       RetryJob.perform_later "CustomCatchError", 6
-      assert_match(/Stopped retrying RetryJob \(Job ID: .*?\) due to a CustomCatchError.*, which reoccurred on \d+ attempts\./, @logger.messages)
     end
+    assert_match(/Stopped retrying RetryJob \(Job ID: .*?\) due to a CustomCatchError.*, which reoccurred on \d+ attempts\./, @logger.messages)
```

Comment:
Moving this assertion out of the block fixes the backtrace that shows if this test fails. Not strictly related to this PR, but this is how I discovered the issue.